src/lib/autoinstall/widgets.rb

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+# Work around YARD inability to link across repos/gems:
+# @!macro [new] seeAbstractWidget
+#   @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FAbstractWidget:${0}
+# @!macro [new] seeCustomWidget
+#   @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FCustomWidget:${0}
+# @!macro [new] seeDialog
+#   @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FDialog:${0}
+# @!macro [new] seeComboBox
+#   @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FComboBox:${0}
```

Comment:
Only macro definitions?

I would have expected that `require "autoinstall/widgets"` would load the `Autoinstall::Widgets` namespace and would give me access to all the classes defined there (similar to `require "autoinstall/presenters"`).

If this is just a collection of yardoc macros it likely does not deserves to be called like the namespace.

Reply:
As discussed _offline_, _widgets.rb_ will be renamed to _yard_doc.rb_ since it is there only for fix a YARD limitation. Also I'll include the clarification about why it works

>  because YARD sorts by filename size(!)

Thanks a lot for the help.